pkgs/development/python-modules/jupytext/default.nix

```
@@ -1,39 +1,47 @@
-{ lib, buildPythonPackage, fetchPypi, isPy27
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, isPy27
+, pytestCheckHook
+, GitPython
+, jupyter_client
 , markdown-it-py
+, mdit-py-plugins
 , nbformat
-, pytest
+, notebook
 , pyyaml
 , toml
 }:
 
 buildPythonPackage rec {
   pname = "jupytext";
-  version = "1.11.0";
+  version = "1.11.2";
 
   disabled = isPy27;
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "9062d001baaa32430fbb94a2c9394ac906db0a58da94e7aa4e414b73fd7d51bc";
+  src = fetchFromGitHub {
+    owner = "mwouts";
+    repo = "jupytext";
```

Opinionated change or my preference :wink:

```suggestion
    repo = pname;
```